package-res/resources/web/pentaho/visual/ccc/categoricalContinuousAbstract/View.js

```diff
@@ -67,6 +67,24 @@ define([
       this.base();
 
       this._configureAxisDisplayUnits(/* isPrimary: */true, "ortho");
+    },
+
+    _createChart: function(ChartClass, options) {
+
+      var chart = this.base(ChartClass, options);
+
+      var visualElemsCountMax = this._getVisualElementsCountMax();
+      if(visualElemsCountMax > 0) {
+        var me = this;
+        chart.override("_onWillCreatePlotPanelScene", function(plotPanel, data, axisSeriesDatas, axisCategDatas) {
```

Comment:
How many times can this be called? (or, in another way, how many PlotPanelScene has a visualization?)